CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ## Master
 
+* Performance optimization for large number of files
```

Comment:
Update description to refer about cleaning of the sandbox dir during installation

Reply:
Not sure I have understood you correctly, please let me know if I need to change this line again.